features/levering-koopsom.feature

```diff
@@ -0,0 +1,152 @@
+Functionaliteit: levering koopsom
+
+    Achtergrond:
+        Gegeven de registratie BRK kent de onroerendezaak met kadastraalonroerendezaakidentificatie: NL.IMKAD.KadastraalObject.76870487970000,
+        En waarvan de volgende velden gevuld zijn: identificatie, type, kadastraleaanduiding, koopsom, koopjaar, aardCultuurbebouwd
+        En de onroerendezaak met kadastraalonroerendezaakidentificatie: NL.IMKAD.KadastraalObject.76870488070000,
+        En waarvan de volgende velden gevuld zijn: identificatie, type, kadastraleaanduiding, koopsom, aardCultuurbebouwd
+
+    Scenario: Onroerendezaak (met koopsom & koopjaar & aardCultuurbebouwd) wordt opgevraagd zonder fields parameter
+        Als onroerendezaak gezocht word met /kadastraalonroerendezaken/NL.IMKAD.KadastraalObject.76870487970000
+        Dan worden de koopsom & koopjaar & aardCultuurBebouwd teruggegeven
+        En wordt de volgende json response teruggegeven:
+        """
+        {
+            "identificatie": "NL.IMKAD.KadastraalObject.76870487970000",
+            "koopsom": {
+                "koopsom": 185000,
+                "koopjaar": 2015
+            },
+            "type": "perceel",
+            "aardCultuurBebouwd": {
+                "code": "11",
+                "waarde": "Wonen"
+            },
+            "kadastraleAanduiding": {
```

Comment:
deze feature moet nog worden aangepast op kadastraleAanduiding als enkele string

Reply:
Aangepast

Reply:
In het eerste scenario is de aanpassing van de Kadastrale Aaanduiding naar 1 string nog niet doorgevoerd. In de rest van de scenario's wel.